.env.required

```
@@ -1,3 +1,8 @@
-APP_NAME
-APP_API_KEY
-APP_PORT
+L9_ENVIRONMENT
```

dotenv-linter (4.0.0)

[warning] [KeyWithoutValue] The L9_ENVIRONMENT key should be with a value or have an equal sign

(KeyWithoutValue)